ps.go

```diff
@@ -32,6 +33,13 @@ var psCommand = cli.Command{
 		if err != nil {
 			return err
 		}
+		status, err := container.Status()
+		if err != nil {
+			return err
+		}
+		if status == libcontainer.Stopped {
+			return fmt.Errorf("cannot ps a container that has run and stopped")
```

Comment:
cannot ps a container that has stopped.

Container can be in stopped status from created status, without running.